src/components/accession2/view/DetailPanel.tsx

```diff
@@ -61,6 +75,19 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
     return strings.OTHER;
   };
 
+  const getCollectionSiteAddress = () => {
+    const city = accession?.collectionSiteCity;
+    const country = accession?.collectionSiteCountryCode;
+    const subdivision = accession?.collectionSiteCountrySubdivision;
+    const data = [
+      city,
+      countries && country && subdivision ? getSubdivisionByCode(countries, country, subdivision)?.name : '',
+      countries && country ? getCountryByCode(countries, country)?.name : '',
```

Comment:
I don't think we need to show the subdivision and country names here. In figma, the example shows CA, USA. Maybe confirm with Ming?

Reply:
I don’t think she was being specific. Doesn’t make sense to show something else that user didn’t see when selecting. The codes are not understood by all. I can remove this for now and create a new bug for Allison and Ming to follow up on.

Reply:
Yes, reasonable!

Reply:
@constanzauanini I will get this in for now - if feedback on https://terraformation.atlassian.net/browse/SW-1822 is different I will follow up accordingly.